packages/components/MIGRATING.md

```diff
@@ -608,6 +610,94 @@ Changes:
 </SegmentedControl.Root>
 ```
 
+### Select
+
+Changes:
+
+-   This component replaces the old `Dropdown` component.
+-   `menuItems` array has been removed and replaced by `children` to follow the composition pattern.
+    -   `children` can have any kind of content (icons, text, etc.) and will be rendered as the content of the item.
+-   `activeItemId` has been replaced by `value`.
+-   `onChange` has been replaced by `onSelect`.
+-   `disabled` has been removed from individual items and moved to the `Root` component.
+    -   If you need to disable per item, you will need to conditionally render the item.
+
+#### Old
+
+```tsx
+const [active, setActive] = useState(args.activeItemId);
+useEffect(() => setActive(args.activeItemId), [args.activeItemId]);
+return (
+    <Dropdown activeItemId={active} onChange={(id) => setActive(id)} menuBlocks={[
+        {
+            id: 'block1',
+            ariaLabel: 'First section',
+            menuItems: [
+                {
+                    id: 1,
+                    title: 'Simple',
+                },
+                {
+                    id: 2,
+                    title: 'Item with icon',
+                    decorator: <IconMusicNote />,
+                    style: MenuItemStyle.Danger,
+                },
+                {
+                    id: 3,
+                    title: 'Item small',
+                    size: MenuItemContentSize.Small,
+                },
+            ],
+        },
+
+    ]}>
+)
+```
+
+#### New
+
+```tsx
+const [activeItem, setActiveItem] = useState('');
+
+return(
+    <Select
+        onSelect={(selectedItem) => {
+            setActiveItem(() => selectedItem);
+        }}
+        value={activeItem}
+    >
+        <Select.Slot name="left">
+            <IconIcon size={16} />
+        </Select.Slot>
+        <Select.Slot name="right">
+            <IconIcon size={16} />
+        </Select.Slot>
+        <Select.Slot name="menu">
+            <Select.Item value="simple">Simple</Select.Item>
+            <Select.Item value="item-with-icon">
+                <div className="tw-flex tw-items-center tw-gap-4">
+                    <IconMusicNote />
+                    <span>Item with icon</span>
+                </div>
+            </Select.Item>
+            <Select.Item value="small-item">
+                <span class="tw-text-sm">Small item</span>
+            </Select.Item>
+        </Select.Slot>
+    </Select>
+)
+
+// Or, for a simpler setup you don't have to use the slots:
+
+<Select>
```

Comment:
A bit strange decision we have made as `Dialog`, `Dropdown`, `Tooltip`, `TextInput` (depending on the usage) and `SegmentedControl` have `.Root` but not the `Select` 🤔

Reply:
personally I prefer the way the `Select` does it. But we should normalize it one way or the other